src/packageurl/contrib/url2purl.py

```diff
@@ -276,3 +276,49 @@ def build_sourceforge_purl(uri):
 @purl_router.route(cargo_pattern)
 def build_cargo_purl(uri):
     return purl_from_pattern('cargo', cargo_pattern, uri)
+
+
+#https://github.com/nexB/scancode-toolkit/tree/develop/plugins/scancode-ctags-macosx_10_9_intel
+@purl_router.route('https?://github.com.*/.*')
```

Comment:
Why is this `github.com.*/` instead of `github.com/`?

Reply:
Got this one :D